components/GymiProviderOverview.tsx

```diff
@@ -2,28 +2,47 @@
 import React, { useState } from "react";
 import Link from 'next/link';
 import { Database } from "@/database.types";
+import {TransformedGymiProviders} from "@/components/UtilityAnalysisInteraction";
 
 // Typen für GymiProviders und CourseDetails
-type GymiProviders = Database['public']['Tables']['GymiProviders']['Row'];
 type CourseDetails = Database['public']['Tables']['CourseDetails']['Row'];
 
+export interface RatedGymiProviders {
+    id: number;
+    provider: string;
+    pricePerformance: number;
+    quality: number;
+    flexibility: number;
+    additionalServices: number;
+    location: number;
+    totalScore: number;
+}
```

Comment:
Here we define a new interface to type the `ratedGymiProviders` since we get this structure of the date as input from `components/UtilityAnalysisInteraction.tsx`